test/metabase/driver/mongo/util_test.clj

```diff
@@ -1,7 +1,8 @@
 (ns metabase.driver.mongo.util-test
   (:require [expectations :refer :all]
             metabase.driver.mongo.util
-            [metabase.test.util :as tu])
+            [metabase.test.util :as tu]
+            [metabase.driver :as driver])
```

Comment:
https://github.com/metabase/metabase/wiki/Metabase-Clojure-Style-Guide#keep-requires-ordered

I thought you had Emacs set up to automatically call `cljr-clean-ns`?